lib/resources/SubscriptionScheduleRevisions.js

```diff
@@ -2,19 +2,8 @@
 
 const StripeResource = require('../StripeResource');
 
-/**
- * SubscriptionScheduleRevisions is a unique resource in that, upon instantiation,
- * requires a subscription schedule id, and therefore each of its methods only
- * require the scheduleId argument.
- *
- * This streamlines the API specifically for the case of accessing a revision
- * on a returned subscription schedule object.
- *
- * E.g. scheduleObject.revisions.retrieve(revisionId)
- * (As opposed to the also-supported stripe.subscriptionSchedules.retrieveRevision(scheduleId,
- * revisionId))
- */
 module.exports = StripeResource.extend({
   path: 'subscription_schedules/{scheduleId}/revisions',
```

Comment:
@rattrayalex-stripe Is it weird that we’re standardizing on leading `/` for methods paths but not for resource paths?

Reply:
IMO we should have leading slashes on both, yes